src/app/frontend/ingress/detail/info.html

```diff
@@ -24,6 +24,9 @@
     <kd-info-card-entry title="[[Namespace|Resource info details section namespace entry.]]">
       {{::$ctrl.ingress.objectMeta.namespace}}
     </kd-info-card-entry>
+    <kd-info-card-entry title="[[Url Spec|Resource url spec for the ingress.]]" ng-repeat="rule in ::$ctrl.ingress.spec.rules">
```

Comment:
Run `gulp build` or `gulp generate-xtbs` to regenerate translations.

Reply:
s/Url/URL